app/src/androidTest/java/org/mozilla/fenix/ui/robots/HistoryRobot.kt

```diff
@@ -166,3 +177,27 @@ private fun snackBarUndoButton() = onView(withId(R.id.snackbar_btn))
 
 private fun assertUndoDeleteSnackBarButton() =
     snackBarUndoButton().check(matches(withText("UNDO")))
+
+private fun deleteAllHistoryPromptTitle() =
+    mDevice
+        .findObject(
+            UiSelector()
+                .textContains(getStringResource(R.string.delete_history_prompt_title))
+                .resourceId("$packageName:id/title")
+        )
+
+private fun deleteAllHistoryPromptSummary() =
+    mDevice
+        .findObject(
+            UiSelector()
+                .textContains(getStringResource(R.string.delete_history_prompt_body))
+                .resourceId("$packageName:id/body")
+        )
+
+private fun deleteAllHistoryPromptEverythingOption() =
```

Comment:
Nit: this could have a shorter, readable name, maybe "deleteHistoryEverythingOption" or even "deleteEverythingOption" while we're in the history robot, and if it's clear enough what it means.

Reply:
Done !